lib/class-wp-duotone-gutenberg.php

```diff
@@ -649,7 +649,7 @@ private static function get_global_styles_presets( $sources ) {
 	private static function get_selector( $block_name ) {
 		$block_type = WP_Block_Type_Registry::get_instance()->get_registered( $block_name );
 
-		if ( $block_type && property_exists( $block_type, 'supports' ) ) {
+		if ( $block_type && $block_type instanceof WP_Block_Type ) {
```

Comment:
The code does not match in WP Core 

https://github.com/WordPress/wordpress-develop/blob/e83f5a1a602b86dbecbf8ee58d6b8761d53b6197/src/wp-includes/class-wp-duotone.php#L956-L958

Reply:
As with the previous code difference, I think the core version needed to be backported to Gutenberg. @spacedmonkey might have further context on why it wasn't.